### PR TITLE
Tickets/dm 25482

### PR DESCRIPTION
**Pull request by iagaponenko**

### CODE REVIEW

File: admin/tools/docker/replication/container/tools/build.sh [hunk 1]
```diff
@@ -34,7 +34,7 @@ docker run \
        -v /etc/group:/etc/group:ro \
        -v $HOME:$HOME \
        -v $PWD:$PWD \
-       qserv/replica:dev_conda \
+       qserv/replica:dev_conda_ceb6bb6 \
```
Comment (jgates108) [inline admin/tools/docker/replication/container/tools/build.sh:37]: Is linking to a specific version long term?

File: core/modules/replica/HttpIngestChunksModule.h [hunk 2]
```diff
@@ -110,7 +110,7 @@ class HttpIngestChunksModule: public HttpModule {
                            std::string const& database,
                            unsigned int chunk) const;
 
-    util::Mutex _ingestManagementMtx;   /// Synchronized access to the Ingest management operations
+    static util::Mutex _ingestManagementMtx;    /// Synchronized access to the chunk management operations
```
Comment (jgates108) [inline core/modules/replica/HttpIngestChunksModule.h:113]: Maybe a few less spaces between the line and the comment.

File: admin/tools/docker/replication/container/tools/build.sh (hunk 1 above)
Comment (iagaponenko) [reply admin/tools/docker/replication/container/tools/build.sh:37]: I hope it's not! It's one of the consequences of the Conda migration of the LSST Stack. The hash in the container tag's name relates to some Qserv-specific overide in the LSST Stack environment:
{code}
LSST_SPLENV_REF=ceb6bb6
{code}
I'm using a specific hash to ensure I have a container based on the corresponding version of Stack.
I also hope this all will get much easier after migrating to the CMake based build system.

Comment (iagaponenko) [reply admin/tools/docker/replication/container/tools/build.sh:37]: And before that, as the LSST Stack will get more stable I may go back to the less elaborate naming convention for naming these containers:
```
qserv/replica:dev
```

File: core/modules/replica/HttpIngestChunksModule.h (hunk 2 above)
Comment (iagaponenko) [reply core/modules/replica/HttpIngestChunksModule.h:113]: 👍

**Pull request merged**